include/caf/events/sensor_data_aggregator_event.h

```diff
@@ -27,9 +27,10 @@ extern "C" {
 struct sensor_data_aggregator_event {
 	struct app_event_header header;
 	const char *sensor_descr;
-	uint8_t *buf;
+	struct sensor_value *samples;
 	enum sensor_state sensor_state;
-	uint8_t sample_cnt;
+	uint8_t sample_cnt;			/* Number of samples. */
```

Comment:
I would not add the comment if member name is self explanatory. It only increases maintenance costs.

However if we want to have a doxygen documentation (maybe we don't want to add it for this header), this should be a proper doxygen comment, plus we need to document the entire structure apart from these two fields. Can be done under a separate commit/PR if neccessary.